makefile-docker.defs

```diff
@@ -117,7 +108,7 @@ test_unit:
 	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm backend pytest -vvv tests/unit
 
 test:
-	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm backend pytest -vvv -m 'not search'
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run  backend pytest -vvv -m 'not search' tests/routes/test_geo_stats.py
```

Comment:
Think the file path has been left in by mistake here?

Reply:
Also, any reason for removing the remove? Seems like keeping it would be good for tidy up. I might be wrong on this though? 🤔

Reply:
oh ffs! ... thanks :grin:

Reply:
ok - debug stuff now removed